metecho/api/models.py

```diff
@@ -1243,6 +1277,14 @@ class ScratchOrg(
     )
     cci_log = models.TextField(blank=True)
 
+    currently_refreshing_dataset_schema = models.BooleanField(default=False)
+    dataset_schema = models.JSONField(
+        default=dict,
+        encoder=DjangoJSONEncoder,
+        blank=True,
+        help_text=_("Cache of the Dev org schema related to this Task"),
+    )
```

Comment:
@jerivas My understanding is that the schema has top-level Objects, each with a label, api_name, and list of nested Fields, each with label and api_name. Is that your understanding, and/or do you have anywhere we can confirm this currently?

If that's the case, I guess I'd expect this to be a list of Objects, not a dict?

Reply:
I'm having a hard time figuring out how this will look. The new function we want to use is [`get_org_schema`](https://github.com/SFDO-Tooling/CumulusCI/blob/67a426763e11c6838c9fa2d518b3da7349ff6a5a/cumulusci/salesforce_api/org_schema.py#L331), which returns a [new `Schema` class](https://github.com/SFDO-Tooling/CumulusCI/blob/67a426763e11c6838c9fa2d518b3da7349ff6a5a/cumulusci/salesforce_api/org_schema.py#L112) but I can't get it to serialize into JSON to store it in the DB or even send it over to the FE 😕

Reply:
Fixed in 69aa94b23dab25b72008896eb3e53fd32ef7283e